website/cue/reference/remap/functions/replace.cue

```diff
@@ -62,11 +62,18 @@ remap: functions: replace: {
 			return: "Pineapples and Bananas"
 		},
 		{
-			title: "Replace with capture groups"
+			title: "Replace with capture groups when not in configuration file"
 			source: #"""
 				replace("foo123bar", r'foo(?P<num>\d+)bar', "$num")
 				"""#
 			return: "123"
 		},
+		{
+			title: "Replace with capture groups only in configuration file (requires escaping `$`)"
```

Comment:
```suggestion
			title: "Replace with capture groups only when set in the configuration file (requires escaping the `$` character)"
```

Reply:
Yeah definitley helps out the clarification! You get 100% on the english exam!